### PR TITLE
NAWQAAN-54

### PR DESCRIPTION
**Pull request by jiwalker-usgs**
Censored values are indicated on plot.

### CODE REVIEW

File: nar_common/static/nar_ui/plots/sample_concentration_plot.js [hunk 1]
```diff
@@ -107,7 +120,6 @@ nar.plots = nar.plots || {};
             legend: {
                    show: false
             },
-            colors:[previousYearsColor, currentYearColor, criteriaLineColor],
             grid:{
```
Comment (mbucknell) [inline nar_common/static/nar_ui/plots/sample_concentration_plot.js]: I like this change. It's clearer as to what color each series uses.

**Comment by mbucknell** [comment]
Good to go :shipit:

**Comment by jiwalker-usgs** [comment]
Thanks, I threw in a formatting commit that shouldn't change any functionality.

**Comment by davidmsibley** [comment]
test

**Pull request merged**